app/assets/javascripts/models/page/project_page.js.coffee

```diff
@@ -88,6 +88,31 @@ ns.ProjectPage = do (ko,
     hasType: =>
       @query.serialize().bounding_box || @query.serialize().polygon || @query.serialize().point
 
+    showTemporal: =>
+      if @query.serialize().temporal
+
+        m_names = new Array("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec")
+        label = "" 
+        dates = @query.serialize().temporal.split(",")
+        
+        dateStart = moment(dates[0].split("T")[0].split("-")).subtract(1, 'months') # months start at '0', so we need to subtract one
```

Comment:
`momentjs` should prevent the need for splitting on `T` and dealing with all of this.

Reply:
Confirmed - nice catch.

Reply:
Actually, sadly, misconfirmed - without the splits it reads the dates incorrectly :(  It appears they are programmatically necessary.